docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,264 @@ tags:
 
 ## Overview
 
-## Available options
-
-Opisać core i plugins i wrzucić linki do reference do obu
-
-Najlepiej jakby to było dostępne też dla wrapperów
-
-## The cascading configuration
+Handsontable uses cascading configuration, a fast way to provide configuration options for the entire table, including its columns and particular cells. To show you how it works, we use the [readOnly](@/api/metaSchema.md#readOnly) and [className](@/api/metaSchema.md#className) options.
 
 ### Entire grid
 
-Contructor
-
-### Single cell
-
-Address by row / col order; A1, B2; ID column and row
+By setting the [readOnly](@/api/metaSchema.md#readOnly) option, we propagate the option through the column settings down to the cells. As a result, all cells have a read-only state.
 
-### Single range of cells
+::: example #example1 --html 1 --css 2 --js 3
+```html
+<div id="example1" class="hot"></div>
+```
+```css
+td.bg-read-only {
+  background-color: #f2f4fb;
+}
+```
+```js
+const container = document.querySelector('#example1');
 
-### Multiple ranges of cells
+const hot = new Handsontable(container, {
+  data: Handsontable.helper.createSpreadsheetData(5, 10),
+  readOnly: true,
+  className: 'bg-read-only',
+  width: 'auto',
+  height: 'auto',
+  rowHeaders: true,
+  colHeaders: true,
+  licenseKey: 'non-commercial-and-evaluation'
+});
+```
+:::
 
 ### Single column
 
-Address by col order; or ID
-
-### Single range of columns
-
-### Multiple ranges of columns
-
-### Single row
-
-Address by order or ID (from column 1)
-
-### Single range of rows
-
-### Multiple ranges of rows
-
-
-
-## Introduction to cell options
-
-Any constructor or column option may be overwritten for a particular cell (row/column combination), using `cell` array passed to the `Handsontable` constructor.
+Moving on, let's set the read-only cells by setting the option for column.
```

Comment:
for a column OR for columns.

Reply:
Fixed within e6b0a2074709e5d8f815b0113f95c1feb9afea54